src/handlers/major_change.rs

```diff
@@ -207,3 +241,21 @@ async fn handle(
     gh_res.context("label setting failed")?;
     Ok(())
 }
+
+fn zulip_topic_from_issue(issue: &PartialIssue) -> String {
```

Comment:
I would prefer to rename the PartialIssue struct and call it something like "ZulipGitHubReference" or something like that -- I don't think we want to be converting it directly from Issue, rather we should be manually constructing it when needed in code. I think the into() is only called once anyway.

Reply:
I renamed, but I think we should keep the `From` impl. It is in fact used twice, and I'd rather have the code in one place. That said, if you would still prefer to inline the From impl and get rid of it, I am still open to it.

Reply:
I think it's fine as a function or so, I just don't particularly like From/Into impls being used in non-generic code.

Reply:
Okay, I converted it to a method on `Issue`. I think I agree that it's better this way :)